.github/workflows/build.yml

```diff
@@ -13,21 +13,24 @@ jobs:
 
     steps:
     - uses: actions/checkout@v3
+    #! -------------------------------------------------------------------------
     - name: dependencies
       run: |
         sudo apt update -y
         sudo apt install -y libgtest-dev libgflags-dev openssl libssl-dev protobuf-compiler protoc-gen-go golang-go cmake
+        sudo apt install -y python3 pylint
+        pip install pytest
 
+        # Needed for Python bindings support
+        sudo apt install -y swig
```

Comment:
Introduce `swig` in CI ... and a collection of changes to invoke `make`, run `pylint` ... etc ...

Basically, bunch of CI integration.